### ISSUE TITLE
consider using uglifyjs for minified version compiler?

### ISSUE THREAD

**Issue by dropwhile** [HenrikJoreteg/ICanHaz.js#9]
https://github.com/mishoo/UglifyJS

**Comment by HenrikJoreteg** [comment]
I'm open to that, can you explain your reasoning?

**Comment by dropwhile** [comment]
Originally my reasoning was simply that I was seeing quite a few projects using uglifyjs, and thus it seemed to be becoming the defacto compressor for javascript/nodejs libraries.

Later, I tried to run uglifyjs on icanhaz's uncompressed version, and while the output was a little smaller than the pre-compressed version (compressed with clojure as i recall, right?), I got odd errors when I tried to use it. I haven't had the gumption to try tracking down exactly what the problem was yet...

So I think unless you are interested in using uglifyjs for reasons other than this ticket (heard about it elsewhere and became interested), then I would have no problem with you closing this ticket as a rejected feature request.

:)
